test/Quaternion.jl

```diff
@@ -171,6 +202,21 @@ end
         end
     end
 
+    @testset "inv does not under/overflow" begin
+        x = 1e300
+        y = inv(x)
```

Comment:
Sorry, I was not correct in the previous review. replacing `1e300` with `1e-300` will fix the test.
```suggestion
        x = 1e-300
        y = inv(x)
```

```julia
julia> x = 1e300
1.0e300

julia> inv(x)
1.0e-300

julia> inv(inv(x))
9.999999999999999e299

julia> inv(inv(inv(x)))
1.0e-300
```